test/console/graphql/queries/observability_queries_test.exs

```diff
@@ -192,6 +196,33 @@ defmodule Console.GraphQl.ObservabilityQueriesTest do
       assert line["log"] == "a log"
     end
 
+    test "it can fetch logs from an elasticsearch index" do
+      user = insert(:user)
+      svc = insert(:service, read_bindings: [%{user_id: user.id}])
+
+      # the index gets set up using a mix task before the test is run, so we can index directly
+      log_document(svc, "valid log message") |> index_doc(@elastic_cluster_url, @elastic_index_name)
+      log_document(svc, "another valid log message") |> index_doc(@elastic_cluster_url, @elastic_index_name)
+      refresh(@elastic_cluster_url, @elastic_index_name)
```

Comment:
I'd move the cluster url into your elasticsearch helper module as a default arg (so users don't need to specify it w/ every call to `index_doc` or `refresh`

Reply:
use default args in `refresh` here too